cms/bundles/utils.py

```diff
@@ -22,3 +32,85 @@ def get_pages_in_active_bundles() -> list[int]:
     return list(
         bundle_page_class.objects.filter(parent__status__in=ACTIVE_BUNDLE_STATUSES).values_list("page", flat=True)
     )
+
+
+def serialize_page(page: "Page") -> dict[str, Any]:
+    """Serializes a page to a dictionary."""
+    return {
+        "id": uuid.uuid4(),
+        "type": "item",
+        "value": {"page": page.pk, "title": "", "description": "", "external_url": ""},
+    }
+
+
+def serialize_preview_page(page: "Page", bundle_id: int, is_previewable: bool) -> dict[str, Any]:
+    specific_page = page.specific_deferred
+    if workflow_state := specific_page.current_workflow_state:
+        state = workflow_state.current_task_state.task.name
+    else:
+        state = "not in a workflow"
+    return {
+        "id": uuid.uuid4(),
+        "type": "item",
+        "value": {
+            "page": None,
+            "title": f"{specific_page.title} ({state})",
+            "description": getattr(specific_page, "summary", ""),
+            "external_url": reverse("bundles:preview", args=[bundle_id, page.pk]) if is_previewable else "#",
+        },
+    }
+
+
+def serialize_bundle_content_for_release_calendar_page(
```

Comment:
I know we try to DRY, but I think in this case we want one "serializer" for the actual publication process, and one for the preview

Reply:
Addressed in [9253a54](https://github.com/ONSdigital/dis-wagtail/pull/217/commits/9253a549c16211183e46856c81d2f90cf3591a5e) – hopefully this is a good compromise while still keeping things DRY when possible.